src/include/simeng/kernel/LinuxProcess.hh

```diff
@@ -95,7 +95,7 @@ class LinuxProcess {
   uint64_t getEntryPoint() const;
 
   /** Get the initial stack pointer address. */
-  uint64_t getStackPointer() const;
```

Comment:
Couldn't this be `getInitialStackPointer()`?

Reply:
Given they have to be called either by `linux_.getInitianStackPointer` or `process_.getInitialStackPointer` I think it will be evident what it refers to

Reply:
Will the referring to the same value change with multithreading?

Reply:
They should do yes, but if this changes then said change should be made in the relevant multi-threading PR